### PR TITLE
update Clair3 to v0.1.11

### PR DESCRIPTION
**Pull request by zhengzhenxian**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/clair3/meta.yaml [hunk 1]
```diff
@@ -1,32 +1,42 @@
...
 requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - cffi
+    - bzip2
+    - xz
+    - zlib
+    - python=3.6.10
```
Comment (bgruening) [inline recipes/clair3/meta.yaml:24]: Is the strict pin for python needed?

Comment (zhengzhenxian) [reply recipes/clair3/meta.yaml:24]: Hi! @[bgruening](https://github.com/bgruening), 

Yes, we need to execute the python script in MakeFile.

Also, I found the CI-built stopped in test_linux for almost 6 hours. I have an old [PR](https://github.com/bioconda/bioconda-recipes/pull/34034) early but failed multiple times, any suggestions on that, thanks!

Comment (bgruening) [reply recipes/clair3/meta.yaml:24]: Using Python is ok, but why do you need this exact version (3.6.10)? Can you remove it? Also do you need this as run-dependency? I only see failed OSX tests, Linux seems to be fine.

Comment (zhengzhenxian) [reply recipes/clair3/meta.yaml:24]: Yes, tensorflow2.2.0 only supports 3.6.10 for python3.6. Python needs to be installed in run-dependency as I need to compile CFFI using ffibuilder in python.
CI-built works now, thanks for your help!

Comment (zhengzhenxian) [reply recipes/clair3/meta.yaml:24]: Yes, as tensorflow2.2.0 only supports 3.6.10 for python3.6. Python needs to be installed in run-dependency as I need to compile CFFI using ffibuilder in python.
CI-built works now, thanks for your help!

**Comment by zhengzhenxian** [comment]
@BiocondaBot please fetch artifacts

**Review by aquaskyline — APPROVED** [review]

**Pull request merged**